test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

```diff
@@ -16,24 +16,22 @@ describe('AuthorisationCaseEvent', () => {
     before(() => {
       nonProd = getAuthorisationCaseEventDefinitions([
         'AuthorisationCaseEvent',
+        'AuthorisationCaseEvent-alternative-service-nonprod',
+        'AuthorisationCaseEvent-alt-service-process-server-nonprod',
         'AuthorisationCaseEvent-amend-court-orders-nonprod',
-        'AuthorisationCaseEvent-deemed-and-dispensed-nonprod',
         'AuthorisationCaseEvent-general-email-nonprod',
         'AuthorisationCaseEvent-general-referral-nonprod',
-        'AuthorisationCaseEvent-alternative-service-nonprod',
-        'AuthorisationCaseEvent-alt-service-process-server-nonprod',
         'AuthorisationCaseEvent-nonprod'
       ]);
 
       allEventsForNonProd = getCaseEventDefinitions([
         'CaseEvent',
+        'CaseEvent-alt-service-process-server-nonprod',
+        'CaseEvent-alternative-service-nonprod',
```

Comment:
petty thing, but can you put them in the same order as in the first array?

Reply:
ok